wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/internal/JsonIntegration.kt

```diff
@@ -171,6 +173,28 @@ abstract class JsonIntegration<F, A> {
     }
   }
 
+  /** Encodes a unsigned value without quotes, like `123`. */
+  private object UnsignedIntAsNumberJsonFormatter : JsonFormatter<Int> {
```

Comment:
wrong PR?

Reply:
hmm, github is really bad with rebasing on merging branch...